config/locales/frontend.en.yml

```
@@ -45,6 +45,10 @@ en:
       confirm:
         title: "Are you sure?"
         text: "This question will be gone forever."
+    mod_mute:
+      confirm:
+        title: "Are you sure?"
+        text: "This will block this user for everyone."
```

would say something like:
```suggestion
        text: "This will mute this anonymous coward for everyone."
```

---

I agree with the mute suggestion here, the anonymous coward part not so much :P